### DISCUSSION TITLE
Crabada ABI

### DISCUSSION THREAD

**Discussion by ksisco** [coccoinomane/crabada.py#87] [Q&A] [answered]
I'm curious how you got the Crabada ABI.   Did you make it by hand or were you able to generate it from a decompile?

**Comment by EdwinLapitan** [comment]
Hey coccoinomane, first I want to thank you for your efforts! I started learning code specifically to learn how to make a bot for crabada but it's turned into a real passion project of mine and I can't get enough. I also have this question and would love to know the answer. Much love from Las Vegas!

**Comment by coccoinomane** [comment, marked as answer]
> I'm curious how you got the Crabada ABI. Did you make it by hand or were you able to generate it from a decompile?

Hi @ksisco !

I got all the ABIs directly from the Crabada's website: you only need to use Chrome dev tools and extract it from the Javascript 😉

Cheers,
Cocco
